### DISCUSSION TITLE
bug: Other records in the same location section is not displayed on page refresh or url link

### DISCUSSION THREAD

**Discussion by salihdev0** [acikkaynak/deprem-yardim-frontend#973] [Ideas]
## Bug Definition
The section where the other records are displayed by id is not displayed on page refresh or when the link is shared.

-- Please add discord contact information
** discord username: @ss.ibrahimbas#4990 **

## Bug environment
Describe the environment produces the bug. For example:
rc.afetharita.com
afetharita.com
## Describe how you are producing the bug step by step

1. Go to [this url](https://rc.afetharita.com/en?lat=37.0546140867465&lng=37.35237836837769&zoom=18&id=549886)
2. Look at the bottom part of the content there should be a 'other records' section
3. Other records are not displayed

## Expected Behaviour
Other records section should be displayed while accesing the website via a link.

## Screen shots

normal usage:
![Screenshot_20230213_094608](https://user-images.githubusercontent.com/76786120/218389754-58ab0875-dd16-4e8b-8d67-31724f03c724.png)

open site with url:
![Screenshot_20230213_094621](https://user-images.githubusercontent.com/76786120/218389778-1fdface5-462f-4a85-bef7-41c2c1372701.png)

## Desktop Information
- Operating System: [Windows 11]
- Browser [chrome]
- Version [unknown]

**Comment by CyhnYldz** [comment]
Should be checked if the location has more than one id to see if there will be others section. the expected and actual results should be specified for the locations if they have one id and more than one id. I will change the status to tast fail to get feed back on this issue.

**Comment by salihdev0** [comment]
When I share a link with you, you will not be able to see other locations on that link.

Just like pulling a value from the database with an id, other data at that location should be queried. Currently, when people share link to each other, they are likely to see missing data.
